File: Accessibility/GetContentDescriptionOverride/src/main/java/net/twisterrob/lints/accessibility/GetContentDescriptionOverrideViolation.java
package net.twisterrob.lints.accessibility;

import android.content.Context;
import android.view.View;

import androidx.annotation.NonNull;

public class GetContentDescriptionOverrideViolation extends View {

	public GetContentDescriptionOverrideViolation(@NonNull Context context) {
		super(context);
	}

	@Override
	public @NonNull CharSequence getContentDescription() {
		return super.getContentDescription();
	}
}

File: Accessibility/GetContentDescriptionOverride/src/main/java/net/twisterrob/lints/accessibility/GetContentDescriptionOverrideViolation.kt
package net.twisterrob.lints.accessibility

import android.content.Context
import android.view.View

class GetContentDescriptionOverrideViolationKt(context: Context) : View(context) {

	@Suppress("RedundantOverride")
	override fun getContentDescription(): CharSequence = super.getContentDescription()
}
